frontend/src/app/task/task-detail-component/task-detail.component.html

```diff
@@ -32,6 +32,9 @@
                 </div>
             </div>
         </oc-icon-data>
+        <oc-icon-data iconName="timelapse" titleKey="i18n.modules.task.dialogs.add.effort.remaining">
+            <p class="task-property">{{getRemainingEffort()}}</p>
```

Comment:
There should be spaces between the curly braces and the statement (Could be wrong in other files too)